template/src/features/counter/counterSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const slice = createSlice({
+  name: 'counter',
+  initialState: {
+    value: 0,
+  },
+  reducers: {
+    increment: state => {
+      state.value += 1;
```

Comment:
Would it be worth adding an explanatory comment here saying that "Redux Toolkit allows us to 'mutate' the state"?